registration/models.py

```diff
@@ -137,8 +137,8 @@ class BetaTestApplication(ValidateModelMixin, TimeStampedModel):
     privacy_policy_url = models.URLField(
         verbose_name='URL to Privacy Policy',
         help_text=('URL to the privacy policy.'),
-        null=True, # to ease migrations
```

Comment:
Just a note here, the reason I asked for this change was to avoid having 2 possible values for *empty* stored on the database (`''` and `null`). :grin: